docs/unumpy_using.rst

```diff
@@ -0,0 +1,35 @@
+.. currentmodule:: unumpy
+
+Using ``unumpy``
+================
+
+``unumpy`` is an in-progress mirror of the NumPy API which allows the user
+to dynamically switch out the backend that they are using. It also allows
+auto-selection of the backend based on the arguments passed into a function.
+
+Currently, only the following functions are supported:
+
+* NumPy `universal functions <https://www.numpy.org/devdocs/reference/ufuncs.html>`_.
```

Comment:
I think this needs a bit more text. I actually read this as "all ufuncs are supported", but after looking at the implementation I think you just mean the `numpy.ufunc` class.

Reply:
All `ufunc`s I found in the NumPy docs ARE supported. 🙂

Reply:
Ah, then I need to read the code better. Can you add "All" at the start of this line?